tests/src/rules/no-unused-modules.js

```diff
@@ -0,0 +1,578 @@
+import { test, testFilePath } from '../utils'
+
+import { RuleTester } from 'eslint'
+import { expect } from 'chai'
+import fs from 'fs'
+
+const getSrc = require( '../../../src/rules/no-unused-modules').getSrc
+const isNodeModule = require( '../../../src/rules/no-unused-modules').isNodeModule
+
+const ruleTester = new RuleTester()
+    , rule = require('rules/no-unused-modules')
+
+const error = message => ({ ruleId: 'no-unused-modules', message })
+
+const missingExportsOptions = [{
+  missingExports: true,
+}]
+
+const unusedExportsOptions = [{
+  unusedExports: true,
+  src: [testFilePath('./no-unused-modules/**/*.js')],
+  ignoreExports: [testFilePath('./no-unused-modules/*ignored*.js')],
+}]
+
+describe('getSrc returns correct source', () => {
```

Comment:
I think if these need unit tests, they should be extracted to a utility/helper file and not exported from the rule itself.

Reply:
I am not sure, if these really need to be tested explicitly, as they are already implicitly tested by the rule tests and a malfunction should usually be captured here. 

What would be the best choice in your opinion? Would you rather remove the explicit tests or extract the functions into a helper module?